community.enforcer/src/main/java/org/kaazing/community/enforcer/rule/RequireSameFile.java

```diff
@@ -24,8 +24,14 @@ public void execute(EnforcerRuleHelper helper) throws EnforcerRuleException {
         MavenProject project;
         try {
             project = (MavenProject) helper.evaluate("${project}");
-            File originalFile = new File(project.getBasedir() + "\\" + originalFilePath);
-            File revisedFile = new File(project.getBasedir() + "\\" + revisedFilePath);
+            File originalFile = new File(originalFilePath);
+            if (!originalFile.exists()) {
+                originalFile = new File(project.getBasedir() + "\\" + originalFilePath);
```

Comment:
Will \\ work on unix and windows? https://docs.oracle.com/javase/7/docs/api/java/io/File.html.  Perhaps use System Properties https://docs.oracle.com/javase/tutorial/essential/environment/sysprop.html file seperator